main.py

```diff
@@ -343,7 +339,7 @@ def on_fail_add_stream(self, err, args):
             (title, text) = args
             QtWidgets.QMessageBox().warning(self, title, text)
         elif err == AddStreamError.DEFAULT_QUALITY_MISSING:
-            (stream_options, url, quality) = args
+            (stream_options, url) = args
```

Comment:
Couldn't really see where the `quality` was used, so I removed it. Let me know if I made a brain-fart @kaszim